lib/ecto/changeset.ex

```diff
@@ -1828,7 +1828,7 @@ defmodule Ecto.Changeset do
     validate_change changeset, field, {:inclusion, data}, fn _, value ->
       if value in data,
         do: [],
-        else: [{field, {message(opts, "is invalid"), [validation: :inclusion]}}]
+        else: [{field, {message(opts, "is invalid"), [validation: :inclusion, enum: data]}}]
```

Comment:
All of these validations use the `Kernel.in/2` macro, which uses `Enum.member?/2` under the hood, so it only works on values that implement the `Enumerable` protocol - hence the key being called `enum`.

`data` could be a potential alternative name to match the parameter to the validator.